packages/react-native-hms/ios/HMSHLSPlayerManager.swift

```diff
@@ -200,6 +200,13 @@ class HMSHLSPlayer: UIView {
         self.addSubview(playerViewController.view)
 
         attachPlayerPlaybackListeners()
+        
+        hmsHLSPlayerViewController?.showsPlaybackControls = false
```

Comment:
is this still configurable from JS side? @ygit

Reply:
No